services/worldstatestore/fake/fake.go

```diff
@@ -227,30 +393,124 @@ func newFakeWorldStateStore(name resource.Name, logger logging.Logger) worldstat
 		streamCtx:               ctx,
 		cancel:                  cancel,
 		logger:                  logger,
+		noise:                   noise,
+		width:                   750,
+		height:                  750,
+		spacing:                 100,
 	}
 
-	fake.initializeStaticTransforms()
+	fake.initializeTransforms()
 	fake.activeBackgroundWorkers.Add(1)
 	go func() {
 		defer fake.activeBackgroundWorkers.Done()
-		fake.animationLoop()
+		fake.animate()
```

Comment:
light rec: call this step() since this is more in line with a physics world than a renderer